### PR TITLE
Update oc.po

### PR DESCRIPTION
**Pull request by Mejans**
Just a small update ;)

### CODE REVIEW

File: locales/oc.po [hunk 1]
```diff
@@ -86,33 +86,35 @@ msgstr "Opcions PNG"
...
 msgid "Number of threads"
-msgstr ""
+msgstr "Nombre de fils d’execucion"
 
 #: yoga_image_optimizer/data/ui/settings-window.glade:56
 msgid ""
 "The number of threads determines the number of images that will be optimized "
 "at the same time. This value should not be higher than your CPU count."
 msgstr ""
+"Lo nombre de fils d’execucion determina lo nombdre d’imatge que serà optimizat "
```
Comment (flozz) [inline locales/oc.po:96]: Is it normal that the word `number` has two different syntax here (`nombre` vs `nombdre`)?

**Comment by flozz** [comment]
I just have an interrogation about the use of both `"nombre"` and `"nombdre"` syntax (but I do not speak occitan so I is maybe normal :) )

Excepting that, everything looks good :)

**Comment by Mejans** [comment]
you are right, it was a typo :)
I'm always on a rush when I translate your software xD Sorry sorry

**Comment by flozz** [comment]
thank you

**Pull request merged**